.gitignore

```diff
@@ -3,7 +3,7 @@
 *.iml
 .env
 .DS_Store
-venv/
+.venv/
```

Comment:
It's a good practice to ignore virtual environment directories like `.venv/`. Ensure that this entry matches the actual name of your virtual environment directory if it's different.